Confidence/src/main/java/com/spotify/confidence/ConfidenceContext.kt

```diff
@@ -19,19 +19,22 @@ interface Contextual : ConfidenceContextProvider {
     /**
      * Add entry to context
      * @param context context to add.
+     * @param skipFetch if true, the context will not initiate a new fetch.
      */
-    fun putContext(context: Map<String, ConfidenceValue>)
```

Comment:
Should we try to align the api structure with the ios one? https://github.com/spotify/confidence-sdk-swift/blob/a4b1e0af5ccfc83d61ca566a78f4a4181b112d63/Sources/Confidence/Confidence.swift#L220